app/views/ShareListView/index.tsx

```diff
@@ -178,32 +229,33 @@ class ShareListView extends React.Component {
 		});
 	};
 
-	// eslint-disable-next-line react/sort-comp
-	internalSetState = (...args) => {
+	internalSetState = (...args: any[]) => {
```

Comment:
It's not an array, it's an object. And since that's the case I think (I'm not sure, that would be something to test), but, what if you just leave it as `any` and then set `this.setState` as `this.setState({ ... args })`?

Reply:
I just changed this any[] to object[], but the TS continues to complain about the array. When we migrate the roomview, that has the same function, we refactor here too!